dask_sql/physical/rel/custom/wrappers.py

```diff
@@ -0,0 +1,497 @@
+        an ``AttributeError`` is raised.
+
+        Parameters
+        ----------
+        X : array-like
+
+        Returns
+        -------
+        transformed : array-like
+        """
+        self._check_method("transform")
+        X = self._check_array(X)
+        output_meta = self.transform_meta
+
+        if isinstance(X, da.Array):
+            if output_meta is None:
+                output_meta = _get_output_dask_ar_meta_for_estimator(
+                    _transform, self._postfit_estimator, X
+                )
+            return X.map_blocks(
+                _transform, estimator=self._postfit_estimator, meta=output_meta
+            )
+        elif isinstance(X, dd._Frame):
+            if output_meta is None:
+                output_meta = _transform(X._meta_nonempty, self._postfit_estimator)
+            try:
+                return X.map_partitions(
+                    _transform,
+                    self._postfit_estimator,
+                    output_meta,
+                    meta=output_meta,
+                )
+            except ValueError:
+                if output_meta is None:
+                    # dask-dataframe relies on dd.core.no_default
+                    # for infering meta
+                    output_meta = dd.core.no_default
+                return X.map_partitions(
+                    _transform, estimator=self._postfit_estimator, meta=output_meta
+                )
+        else:
+            return _transform(X, estimator=self._postfit_estimator)
+
+    def score(self, X, y, compute=True):
+        """Returns the score on the given data.
+
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
+            Input data, where n_samples is the number of samples and
+            n_features is the number of features.
+
+        y : array-like, shape = [n_samples] or [n_samples, n_output], optional
+            Target relative to X for classification or regression;
+            None for unsupervised learning.
+
+        Returns
+        -------
+        score : float
+                return self.estimator.score(X, y)
+        """
+        scoring = self.scoring
+        X = self._check_array(X)
+        y = self._check_array(y)
+
+        if not scoring:
+            if type(self._postfit_estimator).score == sklearn.base.RegressorMixin.score:
+                scoring = "r2"
+            elif (
+                type(self._postfit_estimator).score
+                == sklearn.base.ClassifierMixin.score
+            ):
+                scoring = "accuracy"
+        else:
+            scoring = self.scoring
+
+        if scoring:
+            if not dask.is_dask_collection(X) and not dask.is_dask_collection(y):
+                scorer = sklearn.metrics.get_scorer(scoring)
+            else:
+                # TODO: implement Dask-ML's get_scorer() function
+                # scorer = get_scorer(scoring, compute=compute)
+                pass
```

Comment:
Can you actually raise an error here instead of passing. 
```suggestion
               raise NotImplementedError("Relevant error message")
```